app_dart/lib/src/request_handlers/check_for_waiting_pull_requests_queries.dart

```diff
@@ -61,7 +61,7 @@ query LabeledPullRequestsWithReviews($sOwner: String!, $sName: String!, $sLabelN
             state
           }
         }
-        labels(first: 5) {
+        labels(first: 10) {
```

Comment:
Does this impact the query time?

Reply:
I don't see much difference when trying in the graphql explorer. This makes sense as this just adds more info based on same filters.